### PR TITLE
PKGBUILD: turn ONNX version into a variable

### PR DESCRIPTION
**Pull request by sebalis**
as discussed on https://aur.archlinux.org/packages/obs-backgroundremoval/

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 pkgname=obs-backgroundremoval
 pkgver=0.2.5_beta
-_ONNXRUNTIME_FILE=onnxruntime-linux-x64-1.7.0.tgz
-_onnxdir=onnxruntime-linux-x64-1.7.0
+_onnxversion=1.7.0
+_onnxdir=onnxruntime-linux-x64-${_onnxversion}
```
Comment (PedroHLC) [inline PKGBUILD]: you could call this "_onnxtag" and re-use for multiple purposes...

**Review by PedroHLC — APPROVED** [review]

File: PKGBUILD (hunk 1 above)
Comment (sebalis) [reply PKGBUILD]: Fine with me but I’m not sure what the procedure is – do I now change the file and add a new commit to this pull request?

Comment (PedroHLC) [reply PKGBUILD]: add a new commit to your master branch that it will magically appear here

Comment (sebalis) [reply PKGBUILD]: I have now done the renaming – not sure what you have in mind about reusing.

Comment (PedroHLC) [reply PKGBUILD]: I meant renaming the dir variable and reuse it in the source's url

Comment (sebalis) [reply PKGBUILD]: This form of communication is getting a bit too complicated for me. Can I ask you to just take the file I made and make out of it whatever you like? I really don’t mind if I’m listed as a co-author or not.

**Comment by PedroHLC** [comment]
> This form of communication is getting a bit too complicated for me. Can I ask you to just take the file I made and make out of it whatever you like? I really don’t mind if I’m listed as a co-author or not.

Done, and pushed to AUR, let me know in case newer versions also work.

**Pull request closed without merging**